lib/webview-helpers.js

```diff
@@ -0,0 +1,111 @@
+import _ from 'lodash';
+import logger from './logger';
+import { asyncMap } from 'asyncbox';
+
+const NATIVE_WIN = "NATIVE_APP";
+const WEBVIEW_WIN = "WEBVIEW";
+const WEBVIEW_BASE = WEBVIEW_WIN + "_";
+const CHROMIUM_WIN = "CHROMIUM";
+
+let helpers = {};
+
+// This function gets a list of android system processes and returns ones
+// that look like webviews, with the appropriate webview prefix and their PID
+async function webviewsFromProcs (adb, deviceSocket) {
```

Comment:
@imurchie added some TODOs, moving to adb would require a bit of refactor that I don't have time for at this point, but I agree it should be done in our cleanup phase